server/data/DraftAdjudicationResult.ts

```diff
@@ -14,6 +14,19 @@ export type IncidentStatement = {
   completed?: boolean
 }
 
+export type OffenceRule = {
```

Comment:
I think you are going to have a conflict with Rich's stuff - he has added (some) of this